ProcessLib/NumericsConfig.h

```diff
@@ -129,4 +129,7 @@ static_assert(std::is_same<detail::GlobalMatrixType::IndexType,
 /// GlobalMatrixType::IndexType and the GlobalVectorType::IndexType.
 using GlobalIndexType = detail::GlobalMatrixType::IndexType;
 
+/// A type for scalar number
+using ScalarType = detail::GlobalMatrixType::ScalarType;
```

Comment:
So, if I understand correctly, introduction of the `ScalarType` type is not required for VTK output, I suggest to make a separate PR.